blog/2021/2021-01-07-convos-solves-irc-persistence-problem.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Convos solves IRC's persistence problem
+author: Iván Ávalos
```

Comment:
As we talked about on IRC: If you want to post this on your own web site, then we can add a canonical here to inform the search engines. Like this:

```
title: ...
author: ...
canonical: https//yoursite/blog/whatever
```

Here is an example where we also added a link in the introduction: https://raw.githubusercontent.com/Nordaaker/convos/www.convos.chat/blog/2020/2020-11-21-toroid-org-convos-rocks.md